theme/templates/base.html

```diff
@@ -20,8 +20,6 @@
             <script src="{{ script }}"></script>
             {% endfor %}
         {% endif %}
-        <!-- Mobile nav script -->
-        <script type="text/javascript" src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/scripts/mobile-nav.js"></script>
```

Comment:
Don't delete the mobile menu script. It has nothing to do with the Google analytics functionality.